nodejs/week1/avg.js

```diff
@@ -0,0 +1,26 @@
+const numbersList = process.argv.slice(2);
+
+const numbers = [];
+for (let i = 0; i < numbersList.length; i++) {
+  numbers.push(Number(numbersList[i]));
+}
+
+if (numbers.length === 0) {
+  console.log("Please enter the numbers");
+  process.exit();
```

Comment:
Exit with an error code indicating that something went wrong.
```suggestion
  process.exit(1);
```